packages/react/src/table/docs/overview.mdx

```diff
@@ -286,13 +286,18 @@ You can use custom widths based on the expected length of the data under each co
 
 You can use the `TableHeaderSortable` component to make columns sortable. This component is a wrapper around the `TableHeader` component and adds the necessary aria attributes to make the column sortable.
 
+It is important to add `VisuallyHidden` content to the table’s caption to inform blind screen reader users of the purpose of `TableHeaderSortable` buttons.
```

Comment:
I think this could be worded better. Help?

Reply:
I think it's ok. You probably don't need the `blind` qualifier 🤷 

FWIW I've given some wordy words a shot 👀 

```suggestion
It is important to add `VisuallyHidden` text in the table’s caption to explain the purpose of the `TableHeaderSortable` buttons. This reduces confusion for screen reader users who may navigate past the buttons or not understand their significance when arriving on them.
```

Reply:
That's better. I had blind because sighted SR users can generally see the buttons and the arrows, but it did feel unnecessary How about this?

It is important to add `VisuallyHidden` text in the table’s caption to explain the purpose of the `TableHeaderSortable` buttons. This reduces confusion for screen reader users who may not understand their significance when focusing them.

Reply:
Nice, I think you're on a winner 🙌